templates/services/merge/execution/besu.tmpl

```diff
@@ -24,7 +24,8 @@
     - --sync-mode=X_SNAP
     - --data-storage-format=BONSAI
     - --network={{if .SplittedNetwork}}${EL_NETWORK}{{else}}${NETWORK}{{end}}
```

Comment:
Add `--genesis-file` flag. This flag can't be used with `--network` at the same time

Reply:
Done [8341b27](https://github.com/NethermindEth/sedge/pull/196/commits/8341b27e7c08906845755bfa668a00d602290d25)